ballerina/Module.md

```diff
@@ -1,17 +1,117 @@
+
+3. Provide a name for your integration, and choose the workspace that this integration will have access to.
+
+![integration created](https://github.com/ballerina-platform/module-ballerinax-notion/blob/main/docs/assets/integration-created.png?raw=true)
+
+4. After creating the integration, you will see a page with your Integration Token (which you’ll need in the following steps). Copy the Integration Token and store it securely.
+
+![copy secrets](https://github.com/ballerina-platform/module-ballerinax-notion/blob/main/docs/assets/secret.png?raw=true)
+
+Step 2: Share a Notion Page with Your Integration
+
+1. Go to the Notion page or database that you want your integration to access.
+2. Click on the Share button at the top right corner of the page.
+3. In the sharing options, search for your integration by the name you provided when creating it.
+4. Grant the integration access by selecting it. This allows your integration to read or update the contents of the page or database.
+
+![connect integration](https://github.com/ballerina-platform/module-ballerinax-notion/blob/main/docs/assets/connect-page.png?raw=true)
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+### Step 1: Import the module
+
+Import the `ballerinax/notion` module
+
+```bal
+import ballerinax/notion;
+```
+
+### Step 2: Instantiate a new connector
+
+Create a `notion:ConnectionConfig` with the obtained OAuth2.0 tokens and initialize the connector with it.
+
+```bal
+configurable string authToken = ?;
+
+final notion:Client notion = check new notion:Client(
+    config = {
+        auth: {
+            token: authToken
+        }
+    }
+);
+```
+
+### Step 3: Invoke the connector operation
+
+Now, utilize the available connector operations.
+
+#### Retrieve an existing page
+
+```bal
+PageResponse result = check notion->/v1/pages/["xyz-page-id"];
+```
+
+#### Creating a page inside another page
+
+```bal
+PageBodyParams payload = {
+    "parent": {
+        "type": "page_id",
+        "page_id": testPageId
+    },
+    "properties": {
+        "title": [
+        {
+            "text": {
+            "content": "Test page"
+            }
+        }
+        ]
+    },
+    "icon": {
+        "emoji": "🥬"
+    }
+};
+PageResponse response = check notion->/v1/pages.post(payload);
+```
 
 ## Examples
 
-The `Notion` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/module-ballerinax-notion/tree/main/examples/), covering the following use cases:
+The `ballerinax/notion` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/ballerina-platform/module-ballerinax-notion/tree/main/examples), covering use cases like creating pages, retrieving users, and managing databases.
+
```

Comment:
```suggestion
The `ballerinax/notion` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/ballerina-platform/module-ballerinax-notion/tree/main/examples), covering use cases like creating pages, retrieving users, and managing databases.

1. [Creating a new page](https://github.com/ballerina-platform/module-ballerinax-notion/blob/main/examples/CreateNewPage/main.bal)
2. [Retrieve existing pages](https://github.com/ballerina-platform/module-ballerinax-notion/blob/main/examples/RetrievePage/main.bal)

```

Reply:
Shall we do the same change in `Package.md` and `README.md` files as well?

Reply:
Yep sounds good 👍